open_academy/views/course_views.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <!-- Action of the view configuration -->
+    <record model="ir.actions.act_window" id="course_action_list">
+        <field name="name">Course view</field>
+        <field name="res_model">openacademy.course</field>
+        <field name="view_mode">tree,kanban,form</field>
+    </record>
+
+    <!-- This is tree view configuration -->
+    <record model="ir.ui.view" id="course_view_tree">
+        <field name="model">openacademy.course</field>
+        <field name="name">openacademy.course_view_tree</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="title" />
+                <field name="description"/>
+            </tree>
+        </field>
+    </record>
+
+
+    <!-- Kanban configuration -->
+    <record model="ir.ui.view" id="course_view_kanban">
+        <field name="model">openacademy.course</field>
+        <field name="name">openacademy.course_view_kanban</field>
+        <field name="arch" type="xml">
+            <kanban string="Kanban course" >
+                <templates>
+                    <t t-name="kanban-box">
+                        <div>
+                            <field name="title" />
+                        </div>
+                    </t>
+                </templates>
+            </kanban>
+        </field>
+    </record>
+
+
+
+    <!-- notebook configuration -->
+    <record model="ir.ui.view" id="course_view_notebook">
+        <field name="model">openacademy.course</field>
+        <field name="name">openacademy.course.view_form</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet string="Course">
+                    <notebook>
+                        <!-- Fields as rows -->
+                        <page name="Course information" string="Course information">
+                            <group colspan="10" col="4">
+                                <field name="description" />
+                                <newline /> 
+                                <field name="description" />
+                            </group>
+                        </page>
+                        <!-- Fields as columns -->
+                        <page name="Course information 2" string="Course information 2">
+                            <group col="4">
+                                <field name="description" />
+                                <field name="description" />
+                            </group>
+                        </page>
+                    </notebook>
```

Comment:
Why all this?

The exercise says:

`In the Course form view, put the description field under a tab, such that it will be easier to add other tabs later, containing additional information.
`